src/cli.rs

```diff
@@ -198,6 +198,10 @@ pub struct RootOpts {
     /// Load the OpenSSL legacy provider.
     #[arg(long, env = "VECTOR_OPENSSL_LEGACY_PROVIDER", default_value = "true")]
     pub openssl_legacy_provider: bool,
+
+    /// Disable probing and configuration of root certificate locations on the system for OpenSSL.
```

Comment:
Can you add the full description here and re-run `make generate-component-docs`? That way we can maintain the description in a single location.

```suggestion
    /// Disable probing and configuration of root certificate locations on the system for OpenSSL. 
    /// The probe functionality manipulates the `SSL_CERT_FILE` and `SSL_CERT_DIR` environment variables
    /// in the Vector process. This behavior can be problematic for users of the `exec` source, which by 
    /// default inherits the environment of the Vector process.
```

Reply:
I tried doing that initially, but `make generate-component-docs` didn't do anything to the docs in `website/`.
I think it only operates based on the config schema, which does not include CLI args/envs.
Or there is something else I should be running?

Reply:
My apologies @hhromic!

Reply:
No worries! I will copy it anyway into the in-code comment for posterity.
So whenever it does generate from CLI flags, you will have it there already.